bin/docs-generator.js

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env node
+
+const glob = require('glob');
+const fs = require('fs');
+const path = require('path');
+const pipe = require('@k88/pipe-compose').pipe;
+
+const inlineCommentsRegex = /<!--[\S\s]*-->/;
+const includeRegex = /include\(['"]?([\w\/.]*)['"]?\)/;
```

Comment:
<!-- docs-generator:include` seems more consistentwith the other syntax

Reply:
fair